presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanNodeStatsSummarizer.java

```diff
@@ -148,15 +148,15 @@ private PlanNodeStatsSummarizer() {}
             stats.add(new PlanNodeStats(
                     planNodeId,
                     new Duration(planNodeWallMillis.get(planNodeId), MILLISECONDS),
-                    planNodeInputPositions.get(planNodeId),
-                    succinctDataSize(planNodeInputBytes.get(planNodeId), BYTE),
+                    planNodeInputPositions.getOrDefault(planNodeId, 0L),
```

Comment:
I am not sure if it is the way to go. It looks like you are just hiding the real issue here.

@szymonm Can you please take a look why stats are not calculated properly for index join source?